### PR TITLE
Fixed empty fields in output, output of large arrays, and ip addresses and emails

### PR DESCRIPTION
**Pull request by christodenny**
-If a field in a cert is blank, prints N/A
-Large arrays are now newline separated in the output, instead of all on one line
-Displays cert alternate ip addresses and email addresses as well

@csstaub @alokmenghrajani PTAL

### CODE REVIEW

**Comment by csstaub** [comment]
Instead of having it display N/A, we should just not print that line if a field is empty. Most certs won't have all of the fields.

File: display.go [hunk 1]
```diff
@@ -32,39 +33,87 @@ var layout = `Not Before: {{.NotBefore | certStart}}
...
+	}
+	return str.String()
+}
+
+// displayIpArray formats a given array of ip addresses
+// as a newline separated string to return.
+func displayIpArray(arr []net.IP) string {
+	var str bytes.Buffer
+	for _, elem := range arr {
+		str.WriteString("\n        ")
```
Comment (csstaub) [inline display.go]: I'd prefer a tab here rather than a magic number of spaces

**Comment by christodenny** [comment]
@csstaub updated to use template PTAL

**Comment by csstaub** [comment]
LGTM ✅

**Pull request merged**